### PR TITLE
Add page Signed message standards

### PR DESCRIPTION
**Pull request by blukat29**
## Proposed changes

- Add page references/signed-message describing KIP-97, EIP-191 and EIP-712 standards.
- Fix page learn/computation/debug-tracing for intra-docs link

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [ ] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Review by praveen-klaytn — APPROVED** [review]

File: docs/references/signed-message.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+```js
+kaia.recoverFromMessage('0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', '0x61626364', '0x90824271750d7a09f90a76b6f8ec1e5e2afd31790fea9f43e26c120fef3152be46ad09c76f87bd6c495859fa37127754f1f0780180df53eda80034dac036b8d31b', 'latest')
+
+"0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a"
+```
+
+## EIP-191 Ethereum Signed Message
+
+In Ethereum and several EVM chains, it is widely used practice to prefix the messages with "Ethereum Signed Message" before signing them. For details, please refer to [EIP-191](https://eips.ethereum.org/EIPS/eip-191). Kaia nodes and SDKs support this feature. As with KIP-97, the application has to take care with replay protection when using EIP-191. Nevertheless, using EIP-191 ensures compatibility with other ecosystem tools and streamlines the message handling logic, without the need for EIP/KIP branches.
```
Comment (kjeom) [inline docs/references/signed-message.md:50]: option 1: "Ethereum Signed Message" --> "\x19Ethereum Signed Message:\n" + len(message)

option 2: Ethereum- prefix

File: docs/references/signed-message.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+```js
+kaia.recoverFromMessage('0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', '0x61626364', '0xe67ddbb12ad7c85a28b082bb3f159e637229454d34824bd96c0df38e49bf92d42167ffba7565855585de0c32407b0622b0b66fdfe7bd6566d4a19ca40b39ec631b', 'latest')
+
+"0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a"
+```
+
+## EIP-712 Typed Structured Data
+
+While EIP-191 and KIP-97 were standards for signing a single string, EIP-712 is a standard for signing application data structured in JSON format. This format is more human-readable and efficient to process on EVM. For details, please refer to [EIP-712](https://eips.ethereum.org/EIPS/eip-712). Note that this standard cannot fully prevent replay attacks. As with other standards, applications must take care about replay protection mechanisms.
```
Comment (kjeom) [inline docs/references/signed-message.md:85]: It's better to mention how it works in kaia side.
In the KIP-97 and EIP-191 section, we noticed how kaia node supports from the standard perspective.

Comment (blukat29) [reply docs/references/signed-message.md:85]: AFAIK EIP-712 isn't supported by kaia node nor Kaikas. That's why I kept this section short. dApps can use MetaMask to receive EIP-712 signature anyways.

File: docs/references/signed-message.md (hunk 1 above)
Comment (blukat29) [reply docs/references/signed-message.md]: Fixed [5a75599](https://github.com/kaiachain/kaia-docs/pull/49/commits/5a755998b285a15c7793474ac09543b81225ecfe).

File: docs/references/signed-message.md (hunk 2 above)
Comment (blukat29) [reply docs/references/signed-message.md:85]: Or I can just add MetaMask example (e.g. https://docs.metamask.io/wallet/reference/eth_signtypeddata_v4/, https://docs.metamask.io/wallet/how-to/sign-data/)

Comment (kjeom) [reply docs/references/signed-message.md:85]: @blukat29 Yes, then  It would be better to add the example with the short notice of not supported in kaia.

Comment (blukat29) [reply docs/references/signed-message.md]: Added Kaikas example, and MetaMask docs
https://github.com/kaiachain/kaia-docs/commit/c3b6f67dbc7f82620da37a02f742e408fd720f6c

**Review by kjeom — APPROVED** [review]

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**